--- a/bigtop-packages/src/common/pig/install_pig.sh	2016-05-20 10:42:15.000000000 -0500
+++ b/bigtop-packages/src/common/pig/install_pig.sh	2017-09-07 15:29:10.000000000 -0500
@@ -125,22 +125,28 @@
 #!/bin/bash

 # Autodetect JAVA_HOME if not defined
-. /usr/lib/bigtop-utils/bigtop-detect-javahome
+. \${SNAP}/usr/lib/bigtop-utils/bigtop-detect-javahome

-export HADOOP_PREFIX=\${HADOOP_PREFIX:-/usr/lib/hadoop}
-export HADOOP_CONF_DIR=\${HADOOP_CONF_DIR:-/etc/hadoop/conf}
-export HBASE_HOME=\${HBASE_HOME:-/usr/lib/hbase}
-export HBASE_CONF_DIR=\${HBASE_HOME}/conf
-export ZOOKEEPER_HOME=\${ZOOKEEPER_HOME:-/usr/lib/zookeeper}
-export HIVE_HOME=\${HIVE_HOME:-/usr/lib/hive}
-export HCAT_HOME=\${HCAT_HOME:-/usr/lib/hive-hcatalog}
+if [ -x \${SNAP}/hadoop/usr/bin/hadoop ]; then
+  export HADOOP_PREFIX=\${SNAP}/hadoop/usr
+  export HADOOP_CONF_DIR=\${SNAP}/hadoop/etc/hadoop/conf
+  export PATH=\${SNAP}/hadoop/usr/bin:$PATH
+else
+  export HADOOP_PREFIX=\${HADOOP_PREFIX:-/usr/lib/hadoop}
+  export HADOOP_CONF_DIR=\${HADOOP_CONF_DIR:-/etc/hadoop/conf}
+fi
+export HBASE_HOME=\${SNAP}/hbase/usr/lib/hbase
+export HBASE_CONF_DIR=\${SNAP}/hbase/etc/hbase/conf
+export ZOOKEEPER_HOME=\${SNAP}/zookeeper/usr/lib/zookeeper
+export HIVE_HOME=\${SNAP}/hive/usr/lib/hive
+export HCAT_HOME=\${SNAP}/hive/usr/lib/hive-hcatalog

-exec $INSTALLED_LIB_DIR/bin/pig "\$@"
+exec \${SNAP}$INSTALLED_LIB_DIR/bin/pig "\$@"
 EOF
 chmod 755 $PREFIX/$BIN_DIR/pig

 install -d -m 0755 $PREFIX/$MAN_DIR
-gzip -c pig.1 > $PREFIX/$MAN_DIR/pig.1.gz
+gzip -c ../../bigtop/build/bigtop-packages/src/common/pig/pig.1 > $PREFIX/$MAN_DIR/pig.1.gz

 # Copy in the docs
 install -d -m 0755 $PREFIX/$DOC_DIR
@@ -150,9 +156,9 @@
 # Tutorial
 install -d -m 0755 $PREFIX/$EXAMPLES_DIR
 (cd $BUILD_DIR/tar/pig*/tutorial && tar -cf - .)|(cd $PREFIX/$EXAMPLES_DIR && tar -xf -)
-sed -i -e "s|../build/ivy/lib/Pig|/usr/lib/pig/lib/|" $PREFIX/$EXAMPLES_DIR/build.xml
-sed -i -e "s|<fileset dir=\"..\">|<fileset dir=\"/usr/lib/pig/\">|" $PREFIX/$EXAMPLES_DIR/build.xml
-sed -i -e "s|<path id=\"tutorial.classpath\">|<path id=\"tutorial.classpath\"> <fileset dir=\"/usr/lib/hadoop\"> <include name=\"*.jar\"/> </fileset>|" $PREFIX/$EXAMPLES_DIR/build.xml
+sed -i -e "s|../build/ivy/lib/Pig|\${SNAP}/usr/lib/pig/lib/|" $PREFIX/$EXAMPLES_DIR/build.xml
+sed -i -e "s|<fileset dir=\"..\">|<fileset dir=\"\${SNAP}/usr/lib/pig/\">|" $PREFIX/$EXAMPLES_DIR/build.xml
+sed -i -e "s|<path id=\"tutorial.classpath\">|<path id=\"tutorial.classpath\"> <fileset dir=\"\${SNAP}/hadoop/usr/lib/hadoop\"> <include name=\"*.jar\"/> </fileset>|" $PREFIX/$EXAMPLES_DIR/build.xml

 # Pig log directory
 install -d -m 1777 $PREFIX/var/log/pig
