--- a/bigtop-packages/src/common/pig/do-component-build	2016-05-20 10:42:15.000000000 -0500
+++ b/bigtop-packages/src/common/pig/do-component-build	2017-09-07 15:29:10.000000000 -0500
@@ -18,6 +18,14 @@ set -ex

 . `dirname ${0}`/bigtop.bom

+sed -e '/hbase95.version=/d' \
+    -e '/hive.version=/d' \
+    -e '/zookeeper.version=/d' \
+    -e '/hadoop-common.version=/d' \
+    -e '/hadoop-hdfs.version=/d' \
+    -e '/hadoop-mapreduce.version=/d' \
+    -i ivy/library.properties
+
 cat <<__EOT__ >> ivy/libraries.properties

 hbase95.version=$HBASE_VERSION
@@ -52,13 +60,13 @@ BUILD_OPTS="-Djavac.version=${JDK_VERSIO
 # [ivy:resolve]                :: org.mortbay.jetty#jetty;6.1.26!jetty.zip
 # [ivy:resolve]                ::::::::::::::::::::::::::::::::::::::::::::::
 for h in ${HOME} `mvn help:system | grep '^user.home=' | cut -f2 -d=` ; do
-  rm -rf ${h}/.ivy2/cache/*jetty ${h}/.m2/repository/org/mortbay ${h}/.m2/repository/org/eclipse/jetty
+  rm -rf ${h}/.ivy2/cache/*jetty ${h}/.m2/repository/org/eclipse/jetty
 done


 echo "forrest.maxmemory=256m" >> src/docs/forrest.properties

-export ANT_OPTS="-Xmx4096m"
+export ANT_ARGS="-verbose -debug $ANT_OPTS" ANT_OPTS="-Dmvnrepo=http://repo.maven.apache.org/maven2 $ANT_OPTS -Xmx4096m"
 ant $BUILD_OPTS clean published pigunit-jar smoketests-jar javadoc "$@"
 for build_file in contrib/piggybank/java/build.xml ; do
   ant $BUILD_OPTS -buildfile $build_file clean jar "$@"
