--- a/bigtop-packages/src/common/pig/do-component-build	2016-05-20 10:42:15.000000000 -0500
+++ b/bigtop-packages/src/common/pig/do-component-build	2017-09-07 15:29:10.000000000 -0500
@@ -42,7 +42,7 @@ tar xf  $HOME/.m2/repository/apache-forr
 export FORREST_HOME=`pwd`/apache-forrest-0.9

 BUILD_OPTS="-Djavac.version=${JDK_VERSION} -Dforrest.home=${FORREST_HOME} -Dversion=${PIG_VERSION} -Dhadoopversion=23"
-
+mvn ${ANT_OPTS} -f ivy/pig.pom
 # FIXME: there's a weird synchronization issue between ivy and Maven caches wrt. jetty
 # If we don't do the following the build fails with (note zip instead of jar):
 # [ivy:resolve]                ::::::::::::::::::::::::::::::::::::::::::::::
@@ -58,7 +58,7 @@ done

 echo "forrest.maxmemory=256m" >> src/docs/forrest.properties

-export ANT_OPTS="-Xmx4096m"
+export ANT_ARGS="-verbose -debug $ANT_OPTS" ANT_OPTS="-Dmvnrepo=http://repo.apache.maven.org/maven2 $ANT_OPTS -Xmx4096m"
 ant $BUILD_OPTS clean published pigunit-jar smoketests-jar javadoc "$@"
 for build_file in contrib/piggybank/java/build.xml ; do
   ant $BUILD_OPTS -buildfile $build_file clean jar "$@"
